src/app/loadout/loadout-ui/menu-hooks.tsx

```diff
@@ -103,6 +110,17 @@ export function useLoadoutFilterPills(
     [savedLoadouts]
   );
 
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
What was this complaining about? It seems to have all the right dependencies.

Reply:
See surrounding code -- we feature flag this component internally, so eslint-react thinks these books are called conditionally because it can't see that feature flags are constant. Could probably wrap this component and feature flag in the wrapper, or get rid of the filter pills feature flag.